ml-agents/mlagents/trainers/components/reward_signals/curiosity/model.py

```diff
@@ -17,12 +18,13 @@ def __init__(
         """
         self.encoding_size = encoding_size
         self.policy_model = policy_model
+        self.next_visual_in: List[tf.Tensor] = []
```

Comment:
This is done in create_curiosity_encoders(); just trying to declare all the fields in init.